rails6/en/chapter05-user-products.adoc

```diff
@@ -156,7 +146,7 @@ class User < ApplicationRecord
   has_many :products, dependent: :destroy
 end
 ----
-
+<<<
```

Comment:
It seem to be forgotten conflict token

Reply:
I added it for generate a new page. I tested it with pdf format and its Ok. 

https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/_includes/page-break.adoc

Reply:
Oh sorry I didn't think about it. It sound great!